compiler/src/lnntoamm/Operator.ts

```diff
@@ -51,8 +52,8 @@ class Operator {
       }
       throw new Error(`nope`);
     }
-    const tys = [arg1.ty, ...(arg2 ? [arg2.ty] : [])];
-    return this.fns.filter(fn => fn.acceptsTypes(tys));
+    const tys = [arg1, ...(arg2 ? [arg2] : [])];
+    return Fn.select(this.fns, tys, scope);
```

Comment:
I see for operators it's using the list as originally ordered. Is that correct?

Reply:
yes - as mentioned above, the fn list shouldn't be reversed